Examples/ArcGISToolkitExamples/JobManagerExample.swift

```diff
@@ -309,6 +319,20 @@ class JobManagerExample: TableViewController {
         }
     }
     
+    func startBackgroundTask() -> UIBackgroundTaskIdentifier {
+        var backgroundTaskIdentifier: UIBackgroundTaskIdentifier = .invalid
+        backgroundTaskIdentifier = UIApplication.shared.beginBackgroundTask {
```

Comment:
Can you not just use `let backgroundTaskIdentifier = <call to begin>`? If background isn't supported the `beginBackgroundTask` call will return `.invalid`

Reply:
I wish... I can't because it's used in the beginBackgroundTask block. Not sure how apple designed this api to be used. I found the suggestion above recommended somewhere